RELEASE_NOTES

```diff
@@ -1,3 +1,7 @@
+3.2.1
```

Comment:
this is new api so it needs to be a new minor: `3.3.0`

Reply:
k